### ISSUE TITLE
vlist and elist not completely gagging output

### ISSUE THREAD

**Issue by keneanung** [svof/svof#297]
Title line, delimiters and certain vials are not gagged from the `vlist` and `elist` aliases. Example:

```
Vial                    Venom                          Doses    Months
                                                                        
------------------------------------------------------------------------
Gold-trimmed ceramic vi the venom curare               200      ---
Golden-hued willow vial the venom vardrax              200      ---
                                                                        
------------------------------------------------------------------------

H:3932/3894 M:5455/5459 E:16600/16600 W:21460/21460 exkdb- 11 Scarlatan
737 13:13:02.883
(svof): Elixirs:
  0 an elixir of speed (0s)            1 short
  1 an elixir of mana (199s)           1 short
  1 an elixir of levitation (198s)
  0 an elixir of health (0s)           2 short
  0 an elixir of venom (0s)            2 short
  0 an elixir of frost (0s)            1 short
  0 an elixir of immunity (0s)         2 short
(svof): Salves:
  2 a salve of restoration (278s)
  1 a caloric salve (198s)             1 short
  2 a salve of mass (274s)
  0 an epidermal salve (0s)            2 short
  0 a salve of mending (0s)            2 short
(svof): Empties:
 72 empty (0s)
(svof): Unknowns:
  0 elixir of life enhancement (0s)    2 short
  0 elixir of life enhancemen (0s)     2 short
  1 elixir of the lucky (1s)           1 short
  1 saltwater (87s)
  2 milk (201s)
  1 Cyrenian warpaint (10s)            1 short
  1 deception brew (48s)               1 short
  1 oil (13s)
(svof): Venoms:
 25 the venom aconite (4950s)
 39 the venom curare (7600s)
 12 the venom vardrax (2400s)
  (change desired amounts)
  (append refill request, need 21 refills)
  (refill from tuns, need 21 refills)
```

**Comment by jgh713** [comment]
This is something I was looking into last night. Have a fix to push when I get home. Its in another issue, will mark it and close this later.

**Comment by jgh713** [comment]
Addressed in #67
